resources/user-agents/browsers/uc-browser/uc-web-12.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
This is just a copy of one of the useragents from uc-web-11.json (the Blink based one, as I didn't see any U3 UAs on version 12.0).